File: Programming Language(ver.2)/Go/306.go
package main
import "fmt"

func compute(n int) int {
	if n == 0{
		return 1;
	}else{
		return n * compute(n-1);
	}
}

func main(){
	var n int
	fmt.Scanln(&n)
	fmt.Printf("%d!=%d", n, compute(n))
}

File: Programming Language(ver.2)/C++/306.cpp
#include <iostream>
using namespace std;

int compute(int);
int main()
{
    int n;
    cin >> n;
    cout << n << "!=" << compute(n);
    return (0);
}

int compute(int n)
{
    int i, result = 1;
    for (i = 1; i <= n; i++) {
        result = result * i;
    }
    return result;
}